formats/detailed_guide/publisher/details.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "first_published_at"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "first_published_at": {
+      "type": "string"
+    },
+    "image": {
+      "$ref": "#/definitions/image"
+    },
+    "scheduled_publication": {
+      "type": "string"
+    },
+    "alternative_attachment_email": {
+      "type": "string"
```

Comment:
This should be in the links hash as a link to an organisation, referred to as an `alternative_format_provider_organisation`. We will then expand the `alternative_format_email` from that organisation by default.